<?xml version="1.0"?>
<launch>
  <arg name="camera_namespace" default="hand_cam"/>
  <arg name="serial_no" default="327122075271"/>
  <arg name="depth_fps" default="30"/>
  <arg name="infra_fps" default="30"/> 
  <arg name="color_fps" default="30"/>
  <arg name="depth_width" default="640"/>
  <arg name="depth_height" default="480"/>
  <arg name="infra_width" default="640"/>
  <arg name="infra_height" default="480"/>
  <arg name="color_width" default="1280"/>
  <arg name="color_height" default="720"/>
<include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch">
    <arg name="camera"                value="$(arg camera_namespace)"/>
    <arg name="serial_no"             value="$(arg serial_no)"/>
    <arg name="depth_width"           value="$(arg depth_width)"/>
    <arg name="depth_height"          value="$(arg depth_height)"/>
    <arg name="infra_width"           value="$(arg infra_width)"/>
    <arg name="infra_height"          value="$(arg infra_height)"/>
    <arg name="color_width"           value="$(arg color_width)"/>
    <arg name="color_height"          value="$(arg color_height)"/>
    <arg name="depth_fps"             value="$(arg depth_fps)"/>
    <arg name="infra_fps"             value="$(arg infra_fps)"/>
    <arg name="color_fps"             value="$(arg color_fps)"/>
</include>
</launch>
